src/main/java/seedu/tasklist/ui/AssignmentCard.java

```diff
@@ -173,7 +173,7 @@ public void setStyleToIndicateNonePriority(Label label) {
         styleClass.add(NONE_PRIORITY_STYLE_CLASS);
     }
 
-    public void getDueDate(Label label, Deadline deadline) {
+    public void getDueDate(Label label, Time deadline) {
         LocalDateTime now = LocalDateTime.now();
         LocalDateTime due = LocalDateTime.parse(deadline.value, inputFormat);
         String formattedDue = due.format(DateTimeFormatter.ofPattern("dd MMM yyyy HH:mm"));
```

Comment:
Not really related to your part, but it may be good to use the final static variable `TIME_DATE_AND_TIME_FORMAT` here as well :D

Reply:
I think the `TIME_DATE_AND_TIME_FORMAT` doesn't have the ':' though. @minhhhnguyen2000 are you okay with changing?